### ISSUE TITLE
Allow common numbers for PLR2004

### ISSUE THREAD

**Issue by spacemanspiff2007** [astral-sh/ruff#15902]
### Description

When dealing with bytes it's common do divide by 1024 to get KiByte, MiByte, etc..
Thus this should not trigger PLR2004:

````python
b: bytes = b'asdf'
if len(b) > 1024:
    print('More than 1 KiB')
````

````
ruff.py:3:8: PLR2004 Magic value used in comparison, consider replacing `1024` with a constant variable
  |
2 | b: bytes = b'asdf'
3 | if len(b) > 1024:
  |             ^^^^ PLR2004
4 |     print('More than 1 KiB')
  |
````
